app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,20 @@
       <header class="panel content-header">
         <%= yield :header %>
         <nav class="content-header__actions-right">
+
+          <div class="content-header__roles">
```

Comment:
I like this - ideally it would be nice to have a non-JS fallback implementation, but we've also got other JS dependencies that we don't have time to address.

Reply:
Actually it functions without JS :smile: if you turn off JS it's a dropdown and a 'switch' submit button.  The JS removes the button and makes it autosubmit

Reply:
That's what I get for commenting too early - without going throught the whole code :-)